### PR TITLE
Drop references to sample-repo

### PR DESCRIPTION
**Pull request by Itxaka**
Signed-off-by: Itxaka <igarcia@suse.com>

### CODE REVIEW

**Comment by Itxaka** [comment]
Changed some example to the epinio sample which probably requires being deprecated also? Or we can keep it?
Changed some refs to the c3os repo, which probably would be the one to serve as repo.

Maybe we could change some references to cloudOs by Darren?

File: content/en/docs/Customizing/login.md [hunk 1]
```diff
@@ -13,5 +13,5 @@ By default you can login with the user `root` and `cos` in a vanilla cOS image,
 You can change this by overriding `/system/oem/04_accounting.yaml` in the container image if present, or via [cloud-init](../../reference/cloud_init/#stagesstage_idstep_nameusers).
 
 ### Examples
-- [Changing root password](https://github.com/rancher-sandbox/cos-toolkit-sample-repo/blob/00c0b4abf8225224c1c177f5b3bd818c7b091eaf/packages/sampleOS/build.yaml#L13)
+- [Example accounting file](https://github.com/mudler/c3os/blob/master/files/system/oem/10_accounting.yaml)
 - [Example accounting file](https://github.com/rancher-sandbox/epinio-appliance-demo-sample/blob/master/packages/epinioOS/04_accounting.yaml)
```
Comment (mudler) [inline content/en/docs/Customizing/login.md:17]: The epinio one is archived too, so this needs to be dropped as well

File: content/en/docs/Development/creating_derivatives.md [hunk 2]
```diff
@@ -143,17 +92,7 @@ requires:
...
-
-With luet installed locally and docker running, in your git checkout you can build it also by running `luet build --tree packages system/sampleOS`. This will produce an artifact of `system/sampleOS`. Similary, we could also build separately the sample application with `luet build --tree packages app/sampleOSService`.
-
-The build process by default results in a `build` folder containing the package and the compilation metadata in order to generate a repository.
-
-_Note on reproducibility_: See [the difference between our two samples repositories](https://github.com/rancher-sandbox/epinio-appliance-demo-sample#main-difference-with-cos-toolkit-sample-repo) for an explanation of what are the implications of using a `.luet.yaml` file for building instead of a git submodule.
+The difference is that with `requires` we use the _building_ container that was used to build the packages instead of creating a new image from their results: we are not consuming their artifacts in this case, but the environment used to build them. See also [the luet docs](https://luet-lab.github.io/docs/docs/concepts/packages/specfile/#package-source-image) for more details.
 
 ## Additional packages
```
Comment (mudler) [inline content/en/docs/Development/creating_derivatives.md:97]: This section below should be removed as well, It's still refering to the sample repositories

File: content/en/docs/Reference/_index.md [hunk 3]
```diff
@@ -12,5 +12,4 @@ description: >
 ### Samples repositories
 
 - [Build a derivative and upgrade it with fleet](https://github.com/rancher-sandbox/cos-fleet-upgrades-sample)
-- [Sample repository](https://github.com/rancher-sandbox/cos-toolkit-sample-repo)
 - [EpinioOS sample repository](https://github.com/rancher-sandbox/epinio-appliance-demo-sample)
```
Comment (mudler) [inline content/en/docs/Reference/_index.md:15]: Same here, the epinio appliance is archived as well

File: content/en/docs/Reference/high_level_architecture.md [hunk 4]
```diff
@@ -37,7 +37,7 @@ The fundamental phases can be summarized in the following steps:
 - Add metadata(s) and create a repository
 - (optionally) publish the repository and the artefacts
 
-The developer of the derivative applies a customization layer during build, which is an augmentation layer in the same form of `cos-toolkit` itself. [An example repository is provided](https://github.com/rancher-sandbox/cos-toolkit-sample-repo) that shows how to build a customOS that can be maintained with a container image registry.
+The developer of the derivative applies a customization layer during build, which is an augmentation layer in the same form of `cos-toolkit` itself. [An example repository is provided](https://github.com/rancher-sandbox/epinio-appliance-demo-sample) that shows how to build a customOS that can be maintained with a container image registry.
```
Comment (mudler) [inline content/en/docs/Reference/high_level_architecture.md:40]: Another reference to the epinio one

File: content/en/docs/Reference/selinux.md [hunk 5]
```diff
@@ -7,11 +7,11 @@ description: >
   Build SELinux policies with cOS
 ---
 
-You can use the [cos-toolkit sample](https://github.com/rancher-sandbox/cos-toolkit-sample-repo/tree/master/packages/selinux-policies) policy as a kickstart to customize on top. 
+You can use the [epinio sample](https://github.com/rancher-sandbox/epinio-appliance-demo-sample/tree/master/packages/selinux-policies) policy as a kickstart to customize on top.
```
Comment (mudler) [inline content/en/docs/Reference/selinux.md:10]: This should be removed as well. maybe embed the policies example directly here?

File: content/en/docs/Creating derivatives/build_iso.md [hunk 6]
```diff
@@ -21,7 +21,7 @@ Where `iso.yaml` is the iso specification file, and `--image $IMAGE` is the cont
 
 An example of a yaml file using the cos-toolkit opensuse repositories and syslinux:
 
-{{<githubembed repo="rancher-sandbox/cos-toolkit-sample-repo" file="iso.yaml" lang="yaml">}}
+{{<githubembed repo="mudler/c3os" file="iso.yaml" lang="yaml">}}
```
Comment (mudler) [inline content/en/docs/Creating derivatives/build_iso.md:24]: maybe we should provide an iso.yaml example file inside cos-toolkit `example` dir

**Review by mudler — CHANGES_REQUESTED** [review]
Just few nits, mainly because there are still references to the epinio appliance which is archived too

**Comment by Itxaka** [comment]
redy to re-review @mudler

File: content/en/docs/Customizing/upgrades.md [hunk 7]
```diff
@@ -38,7 +38,6 @@ Specifically, it allows to configure:
 - **VERIFY**: Turns on and off image verification. Currently available for official `cOS` release channels
 - **RECOVERY_IMAGE**: Allows to specify a different image for the recovery partition. Similarly to **UPGRADE_IMAGE** needs to be either an image reference or a package.
 
-An example on how to tweak this file via cloud-init is available [here](https://github.com/rancher-sandbox/cos-toolkit-sample-repo/blob/7355876847367b75485873987e1217f1e1fe6254/packages/sampleOS/02_upgrades.yaml#L41)
```
Comment (mudler) [inline content/en/docs/Customizing/upgrades.md:41]: We can link directly to the cOS ones here https://github.com/rancher-sandbox/cOS-toolkit/blob/master/packages/cloud-config/oem/02_upgrades.yaml

Comment (mudler) [reply content/en/docs/Customizing/upgrades.md:41]: ah. was already linked above, nevermind!

**Review by mudler — APPROVED** [review]

**Pull request merged**